libs/image/include/image/ColorTransform.h

```diff
@@ -231,8 +230,7 @@ inline std::unique_ptr<uint8_t[]> fromLinearToRGB_10_11_11_REV(const LinearImage
     using namespace filament::math;
     size_t w = image.getWidth();
     size_t h = image.getHeight();
-    UTILS_UNUSED_IN_RELEASE size_t channels = image.getChannels();
-    assert(channels >= 3);
+    assert(image.getChannels() >= 3);
```

Comment:
In `fromLinearToGrayscale` below, they already made the choice of moving the function call inside the `assert`. I think it's ok to have it like this, no need for a local variable. The debugger can see the number of channels inside `image` anyway, so debugging experience is still _stellar_.